lib/metadata/linux/LinuxSystemd.rb

```diff
@@ -130,7 +130,11 @@ def service_targets_xml(service)
     end
 
     def duplicated_link?(file)
-      existing_dirs.any? { |dir| @fs.getLinkPath(file).start_with?(dir) } if @fs.fileSymLink?(file)
```

Comment:
I think it would be easier and faster just to change this to:
```ruby
existing_dirs.any? { |dir| @fs.getLinkPath(file).start_with?("#{dir}/") } if @fs.fileSymLink?(file)
```

Reply:
Changed.